caffe2/ideep/operators/quantization/int8_add_op.cc

```diff
@@ -0,0 +1,73 @@
+#include <caffe2/ideep/ideep_utils.h>
+
+namespace caffe2 {
+
+template <bool ReluFused>
```

Comment:
Please put this class definition in anonymous namespace. Same for all the classes here.

Reply:
will fix it soon